src/libraries/System.Drawing.Common/src/System/Drawing/Image.cs

```diff
@@ -153,6 +153,15 @@ public void Dispose()
         /// </summary>
         public void Save(string filename) => Save(filename, RawFormat);
 
+        private static void CheckDirectoryExists(string filename)
```

Comment:
Nit: we usually name these methods, `ThrowIfDirectoryDoesntExist` so that when you're reading call sites, you know that method is going to throw if it doesn't exist.

Reply:
Awesome. Thanks for the feedback. That was my first choice, but other calls were calling CheckStatus(), which actually does a throw, so I tried to align my renaming with that. Do you have any issue with me renaming that method to ThrowIfStatusNotOk(), as well as this one?

Reply:
Nope, I don't mind at all, it is always good to be consistent.

Reply:
After checking more closely, there are 590 calls to CheckStatus(), spread amongst 36 files. Given the scope of that change, it probably makes sense to do that in a different issue/PR

Reply:
Yeah I was going to mention that as well 😄